src/test/java/com/microsoft/sqlserver/jdbc/bulkCopy/bulkCopyTest.java

```
@@ -0,0 +1,181 @@
+        		break;
+        	
+        	case java.sql.Types.DECIMAL:
+        	case java.sql.Types.NUMERIC:
+        		assertTrue(0 == (((BigDecimal) srcValue).compareTo((BigDecimal) dstValue)) , "Unexpected decimal/numeric/money/smallmoney value");
+        		break;
+        		
+        	case java.sql.Types.DOUBLE:
+        		assertTrue(
+        				(((Double) srcValue).doubleValue() == ((Double) dstValue).doubleValue()), "Unexpected float value");
+        		break;
+        	
+        	case java.sql.Types.REAL:
+        		assertTrue((((Float) srcValue).floatValue() == ((Float) dstValue).floatValue()), "Unexpected real value");
+        		break;
+        	
+        	case java.sql.Types.VARCHAR:
+        	case java.sql.Types.NVARCHAR:
+        		assertTrue(((((String) srcValue).trim()).equals(((String) dstValue).trim())), "Unexpected varchar/nvarchar value ");
+        		break;
+        		
+        	case java.sql.Types.CHAR:
+        	case java.sql.Types.NCHAR:
+        		assertTrue((((String) srcValue).equals(((String) dstValue))), "Unexpected char/nchar value ");
+        		break;
+        	
+        	case java.sql.Types.TIMESTAMP:
+        		assertTrue((((Timestamp) srcValue).getTime() == (((Timestamp) dstValue).getTime())), "Unexpected datetime/smalldatetime/datetime2 value");
+        		break;
+
+        	case java.sql.Types.DATE:
+        		assertTrue((((Date) srcValue).getTime() == (((Date) dstValue).getTime())), "Unexpected datetime value");
+        		break;
+        		
+        	case java.sql.Types.TIME:
+        		assertTrue(((Time) srcValue).getTime() == ((Time) dstValue).getTime(), "Unexpected time value ");
+        		break;
+        		
+        	case microsoft.sql.Types.DATETIMEOFFSET:
+        		assertTrue( 0 == ((microsoft.sql.DateTimeOffset) srcValue).compareTo((microsoft.sql.DateTimeOffset) dstValue), "Unexpected time value ");
+        		break;
+        		
+        	default:
+        		System.out.println("Unknow type "+destMeta.getColumnType(i));
```

should we throw exception if there is a missing type?